android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java

```diff
@@ -377,6 +378,7 @@ public void setLoadingScreenManifest(final JSONObject manifest) {
       ExperienceActivityUtils.setNavigationBar(manifest, ExperienceActivity.this);
 
       showLoadingScreen(manifest);
+      mLoadingProgressPopupController.show();
```

Comment:
should we only do this in development mode / when loading progress is sent?

Reply:
I'll double check existing codebase and how it's handled right now and if it's not designed the way you're seeing it here I'll convert it to that way, cause it seems reasonable 💪

Reply:
Yeah, I've confirmed that on the current `master` this `LoadingProgressBar` is not shown for `home` when in development mode 😞 I'm going to change it 💪